playbooks/roles/os_temps/tasks/check_new_app.yml

```diff
@@ -25,12 +25,12 @@
 
 # Setup tag if current container image build is successful
 - name: Modify tags on images
-  shell: "{{ oc_bin }} get imagestream | awk '{print $2}' | grep -v DOCKER | sed 's/.*5000\\///g' | grep '{{ build_config_name_files[template_name] }}' | xargs -i {{ oc_bin }} tag {}:latest {}:{{ tag }}"
+  shell: "{{ oc_bin }} get imagestream | awk '{print $2}' | grep -v DOCKER | sed 's/.*5000\\///g' | grep '{{ build_config_name_files[template_name] }}' | xargs -I{} {{ oc_bin }} tag {}:latest {}:{{ tag }}"
```

Comment:
Do we need to change this to -I vs -i?

Reply:
Oh whoops, i meant to leave a comment on this. `-i` is deprecated and `-I` is the preferred option. Also, `-i` does not exist for the bsd xargs command.

http://man7.org/linux/man-pages/man1/xargs.1.html
```
-i[replace-str], --replace[=replace-str]
              This option is a synonym for -Ireplace-str if replace-str is
              specified.  If the replace-str argument is missing, the effect
              is the same as -I{}.  This option is deprecated; use -I
              instead.
```

Reply:
Does this address all instances of xargs -i?

Reply:
I went through and changed all of them from `-i` to `-I{}`, as according to the man page they have the effect